musixmatch/album_tracks.json

```diff
@@ -15,9 +15,9 @@
             "page_size": false
         },
         "pagination": {
-            "type": "offset",
-            "offsetKey": "offset",
-            "limitKey": "limit",
+            "type": "page",
```

Comment:
Is this because we previously don't have page pagination? But musixmatch is actually using the page pagination?

Reply:
Yes! There was a mistake with the pagination before because these values were incorrect and the count parameter wouldn't work properly.